make_station_map_tests.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from unittest.mock import MagicMock
+import pandas as pd
+
+from source.make_station_map import get_and_format_trip_data
+
+class ScriptTests(unittest.TestCase):
+    def test__get_and_format_trip_data__always__returns_correct(self):
+        dummy_bike_trip_data_frame = pd.DataFrame(
+            [[1,  ' 2018-06-30 23:59:55', '2018-07-01 00:38:10', 1, 2295, 3, ' Spaulding Ave & Armitage Ave', 340, ' Clark St & Wrightwood Ave', 'Subscriber', 'Male', 1989],
+             [2, ' 2018-06-30 23:59:56', '2018-07-01 00:17:14', 2, 1038, 4, ' Spaulding Ave & Armitage Ave', 338, 'Calumet Ave & 18th St', 'Subscriber', 'Female', 1985],
+             [3,    ' 2018-06-30 23:59:48', '2018-07-01 00:08:45', 3, 537, 5, ' Clark St & Wrightwood Ave', 506, ' Spaulding Ave & Armitage Ave', 'Subscriber', 'Male', 1980]],
+            columns=['trip_id', 'start_time', 'end_time', 'bikeid', 'tripduration',
+       'from_station_id', 'from_station_name', 'to_station_id',
+       'to_station_name', 'usertype', 'gender', 'birthyear']
+        ).astype(dtype={
+            "trip_id": "int64",
+            "start_time": "datetime",
```

Comment:
Actually, for this dataframe, since it's the one read from the file initially, `start_time` and `end_time` should not be of this datatype, I'd think probably `string`. Without this, we're not really testing that the convert steps are occurring in this function (I could remove two lines from the method and this test would still run)

I'd recommend actually reading in the file and then inspecting it's elements in the REPL.